src/py42/clients/settings/device_settings.py

```diff
@@ -0,0 +1,415 @@
+from py42._compat import str
+from py42._compat import UserDict
+from py42._compat import UserList
+from py42.clients.settings import SettingProperty
+from py42.clients.settings import show_change
+from py42.clients.settings._converters import bool_to_str
+from py42.clients.settings._converters import days_to_minutes
+from py42.clients.settings._converters import minutes_to_days
+from py42.clients.settings._converters import str_to_bool
+from py42.exceptions import Py42Error
+
+
+class DeviceSettingsDefaults(UserDict, object):
+    """Class used for managing an Organization's Device Default settings. Also acts as a
+    base class for `DeviceSettings` to manage individual device settings."""
+
+    def __init__(self, device_dict, org_settings):
+        self.data = device_dict
+        self._org_settings = org_settings
+        self.changes = org_settings.changes
+        self._destinations = org_settings.data["settings"]["destinations"]
+        self.data["settings"] = {
+            "serviceBackupConfig": self.data["serviceBackupConfig"]
+        }
+        self.backup_sets = [
+            BackupSet(self, self.changes, backup_set)
+            for backup_set in self.data["serviceBackupConfig"]["backupConfig"][
+                "backupSets"
+            ]
+        ]
+
+    @property
+    def available_destinations(self):
+        """Returns a dict of destinations available to be used by devices. Dict keys are
+        destination guids and values are destination names.
+        """
+        return {d["guid"]: d["destinationName"] for d in self._destinations}
+
+    warning_email_enabled = SettingProperty(
```

Comment:
Do these properties show up in the doc strings?

Reply:
The docstrings below each property do get rendered correctly in the sphinx docs. But I just checked running `help()` on a `DeviceSettings` instance in the repl and it shows the docstring of the `SettingProperty` class there instead.. I wonder if there's any way to change that.